app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/MessageComposer.kt

```diff
@@ -341,13 +368,20 @@ private fun MessageComposeActions() {
 }
 
 @Composable
-private fun AdditionalOptionButton() {
+private fun AdditionalOptionButton(messageComposerInnerState: MessageComposerInnerState) {
```

Comment:
in my opinion instead of passing messageComposerInnerState: MessageComposerInnerState as a whole it would make more sense to pass booleans, that way we can see from function declaration directly what it can do .

Reply:
I'll leave it as it is right now, as it's the straight forward way to mutate the inner state from inside the button, without having to pass a lambda to it. If I see a window of time, I'll give a look in the next PR's for this component

Reply:
Pls add TODO then so we do not forget

Reply:
Ended up implementing solution passing the lambda and the boolean =)